lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -238,6 +243,11 @@ def setUp(self):
         self.org = u"ꜱᴛᴀʀᴋ ɪɴᴅᴜꜱᴛʀɪᴇꜱ"
         self.org_html = "<p>'+Stark/Industries+'</p>"
 
+        self.request = self.request_factory.get("foo")
+        self.request.user = self.user
+
+        self.course = modulestore().get_course(self.course.id)
```

Comment:
Perhaps worth a comment here to say something like "refresh course object from the store"?